Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/common/taimaba/TaimabaAntispam.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
If this class is exactly the same as VichanAntiSpam, then there's no reason to have it at all. I can already tell that this is old code anyways, as it has an OkHttpClient instance, which should be dependency injected now.